### PR TITLE
port to 2.0

### PR DESCRIPTION
**Pull request by notwaldorf**
Straight forward hybrid port

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -18,16 +18,35 @@
...
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#2.0-preview",
+    "iron-test-helpers": "PolymerElements/iron-test-helpers#2.0-preview",
+    "paper-button": "PolymerElements/paper-button#2.0-preview",
+    "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
+    "web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.6",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.1"
   },
-  "ignore": []
+  "ignore": [],
+  "resolutions": {
```
Comment (valdrinkoshi) [inline bower.json]: remove `resolutions`

File: demo/index.html [hunk 2]
```diff
@@ -19,20 +19,57 @@
...
+    <p><b>Note</b>: in order to hear the announcements, be sure to turn on your favorite screen reader!</p>
+
+    <h4>Announcer used directly on the main document</h4>
+    <demo-snippet>
+      <template>
+        <input id="input" value="Pizza is delicious">
+        <paper-button raised onclick="_announce()">Announce</paper-button>
+
+        <script>
+          document.addEventListener('HTMLImportsLoaded', function() {
```
Comment (valdrinkoshi) [inline demo/index.html]: It's better to listen for `WebComponentsReady` on `window`

File: test/index.html [hunk 3]
```diff
@@ -9,16 +9,14 @@
...
     <meta charset="utf-8">
     <title>iron-a11y-announcer tests</title>
-    <script src="../../webcomponentsjs/webcomponents.js"></script>
     <script src="../../web-component-tester/browser.js"></script>
   </head>
   <body>
     <script>
       WCT.loadSuites([
-        'iron-a11y-announcer.html',
+        'iron-a11y-announcer.html?wc-shadydom=true&wc-register=true',
```
Comment (valdrinkoshi) [inline test/index.html]: use `wc-ce=true`

**Review by valdrinkoshi — COMMENTED** [review]
LGTM, minor things

File: demo/index.html (hunk 2 above)
Comment (notwaldorf) [reply demo/index.html]: ooo yeah, this is ancient

**Comment by notwaldorf** [comment]
@valdrinkoshi used the right `wc-ce` flag in tests, added back `iron-component-page`

**Pull request merged**